scripts/Text Grammar Corrector/README.md

```diff
@@ -0,0 +1,18 @@
+# Grammar Bot
+
+## About :
+
+---
+
+- This script takes an englsh text paragraph input. Using the language API, it identifies the mistakes in the paragraph. The script then uses the mistakes dictionary and returns the updated paragraph.
+
+---
+
+## How to use it :
```

Comment:
hi @rajatkeshri  please mention the detailed process to install requirements and run the python file. mention the command to install and run the file

Reply:
Updated